galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -353,3 +353,16 @@ messages:
         type: string
       - name: dupno
         type: string
+
+  - name: "VirtualServiceHostNotFoundInGateway"
+    code: IST0132
+    level: Error
```

Comment:
I would call this a warning

Reply:
make sense, since we can create `VirtualService` first before `Gateway`.

Reply:
this is done